zerver/lib/actions.py

```diff
@@ -2883,7 +2883,9 @@ def notify_realm_emoji(realm):
     send_event(event, user_ids)
 
 def do_add_realm_emoji(realm, name, img_url):
-    RealmEmoji(realm=realm, name=name, img_url=img_url).save()
+    emoji = RealmEmoji(realm=realm, name=name, img_url=img_url)
+    emoji.full_clean()
```

Comment:
I have not enabled URL checks as this is wasting resources on our side and I think that the user should make sure the URL he adds is the right one and it loads. If you think otherwise, please let me know and I'll enable this check.